### ISSUE TITLE
Plugin conflicts with vim-sneak and vim-cool

### ISSUE THREAD

**Issue by ranebrown** [kkoomen/vim-doge#58]
I ran into conflicts with a couple of plugins that interfere with the tab/s-tab mappings to highlight the next TODO entry and replace the selected text.
https://github.com/romainl/vim-cool
https://github.com/justinmk/vim-sneak

Using vim-sneak with the below mappings prevents the highlighted text from being replaced if any of the mapped characters are pressed.

```vim
map f <Plug>Sneak_f
map F <Plug>Sneak_F
map t <Plug>Sneak_t
map T <Plug>Sneak_T
```

Using vim-cool causes the next TODO item to not be visually selected but instead inserts the cursor directly in front of the TODO block.

I realize these aren't issues with vim-doge but thought it might be useful to post this info.

**Comment by kkoomen** [comment]
> Using vim-sneak with the below mappings prevents the highlighted text from being replaced if any of the mapped characters are pressed.

That'd be logical, because you always map a specific character at any time, unless DoGe, where we only add/remove the mappings when you need it (which is during the generation of the comment). You can be more specific with your mappings by specifying it with the mode. I think it's enough for you to just map this for normal mode only using `nmap` instead of `map`.

> Using vim-cool causes the next TODO item to not be visually selected but instead inserts the cursor directly in front of the TODO block.

The reason this happens is because vim-cool will remove the search highlighting and when you press `<Tab>` it will search for the next TODO within the comment, so  this indeed conflicts. I will discuss with @romainl for a possible solution.

**Comment by kkoomen** [comment]
This bug has been fixed and released in [v1.15.3](https://github.com/kkoomen/vim-doge/tree/v1.15.3).

Feel free to submit any new issues if you experience any unwanted behavior in the future. Thanks for your contribution.
